api/app/models/fetch/predictions.py

```diff
@@ -182,7 +181,7 @@ async def _fetch_model_predictions_by_station_codes(model: ModelEnum, station_co
     """ Fetch predictions from database.
     """
     # Using the list of station codes, fetch the stations:
-    stations = await app.stations.get_stations_by_codes(station_codes)
+    stations = await get_stations_by_codes(station_codes)
```

Comment:
please revert this bit.
```suggestion
    stations = await app.stations.get_stations_by_codes(station_codes)
```